ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -446,36 +449,133 @@ def test_full_backup_reinstall_restore_with_vault(self):
 
 
 class TestBackupAndRestoreWithReplica(IntegrationTest):
-    """Regression test for https://pagure.io/freeipa/issue/7234"""
-    num_replicas = 1
+    """
+        Regression tests for issues 7234 and 7455
```

Comment:
Please use proper docstring indention style. The title line should be on the same line as triple quotes. One empty line between header line and body. The rest of the body should have the same indention as the triple quote block.

```
    """Regression tests for issues 7234 and 7455

    https://pagure.io/freeipa/issue/7234
        - check that oddjobd service is started after restore
        - check new replica  setup after restore
    https://pagure.io/freeipa/issue/7455
        check that after restore and replication reinitialization
            - users and CA data are at state before backup
            - CA can be installed on existing replica
            - new replica with CA can be setup
    """
```

Reply:
fixed